brapi.yaml

```diff
@@ -1181,6 +1181,391 @@ paths:
+      parameters:
+      - name: methodDbId
+        in: path
+        description: Id of the method to retrieve details of.
+        required: true
+        style: simple
+        explode: false
+        schema:
+          type: string
+      - name: Authorization
+        in: header
+        description: "HTTP HEADER - Token used for Authorization \n\n<strong>Bearer\
+          \ {token_string} </strong>"
+        required: false
+        style: simple
+        explode: false
+        schema:
+          pattern: ^Bearer .*$
+          type: string
+        example: Bearer XXXX
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MethodResponse'
+              example:
+                metadata:
+                  datafiles: []
+                  pagination:
+                    currentPage: 0
+                    pageSize: 0
+                    totalCount: 0
+                    totalPages: 0
+                  status: []
+                result:
+                  class: Numeric
+                  description: Standard rolled measuring tape
+                  formula: a^2 + b^2 = c^2
+                  methodDbId: m1
+                  methodName: Tape Measure
+                  name: Tape Measure
+                  ontologyReference:
+                    documentationLinks:
+                    - URL: https://ontology.org/m1
+                      type: RDF
+                      url: https://ontology.org/m1
+                    ontologyDbId: MO_123
+                    ontologyName: Ontology.org
+                    version: "17"
+                  reference: google.com
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                type: string
+              example: |-
+                ERROR - 2018-10-08T20:15:11Z - Malformed JSON Request Object
+                ERROR - 2018-10-08T20:15:11Z - Invalid query parameter
+                ERROR - 2018-10-08T20:15:11Z - Required parameter is missing
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - Missing or expired authorization
+                token
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - User does not have permission
+                to perform this action
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - The requested object DbId is
+                not found
+    put:
```

Comment:
I didn't see any code for updating the database, so this doesn't appear to be supported.